packages/block-editor/src/components/block-tools/style.scss

```diff
@@ -57,6 +57,19 @@
 	}
 }
 
+// The black plus that shows up on the right side of an empty paragraph block,
+// or the initial appender that exists only on empty documents.
+.block-editor-block-list__empty-block-inserter.block-editor-block-list__empty-block-inserter {
```

Comment:
> the initial appender that exists only on empty documents.

In an empty document the classnames in use for the default appender appear to be `.block-editor-default-block-appender .block-editor-inserter`. Was that rule removed intentionally when moving to this separate file?

Also, it looks like the markup for the default block appender is rendered inside the iframe rather than outside it — so we might still need that rule in `content.scss`, unless there's another way to go about this. Here's where the default appender lives:

https://github.com/WordPress/gutenberg/blob/f699d2b87292cd2fb3898f8814c337e7442d8e03/packages/block-editor/src/components/default-block-appender/index.js#L94-L99